### PR TITLE
clusterloggingforwarding package added to the clusterlogging

### PR DESCRIPTION
**Pull request by elenagerman**
clusterloggingforwarding package added to the clusterlogging

### CODE REVIEW

File: pkg/olm/clusterserviceversion.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package olm
 import (
 	"context"
 	"fmt"
+
```
Comment (trewest) [inline pkg/olm/clusterserviceversion.go:6]: Accident?

**Review by trewest — CHANGES_REQUESTED** [review]
Just one nit, otherwise looks good

File: pkg/olm/clusterserviceversion.go (hunk 1 above)
Comment (elenagerman) [reply pkg/olm/clusterserviceversion.go:6]: no, there is no explanation, but lint verification fails on fmt for clusterserviceversion, so I have to fix it each time

**Review by trewest — APPROVED** [review]
/lgtm

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**